### PR TITLE
[README] clarify spec changes in ratification process

### PR DESCRIPTION
**Pull request by mkrainiuk**
# Description

PR adds some clarification about oneAPI spec related changes in the RFC ratification process.
The need for the clarification was discussed in https://github.com/uxlfoundation/oneAPI-spec/pull/561#discussion_r1765080857

# Checklist

## All Submissions

- [x] Do all unit tests pass locally? Attach a log. N/A
- [x] Have you formatted the code using clang-format? N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ contributors as reviewers to the PR.
...
     * It's recommended to keep all feedback as part of PR review, so it also
 will be documented in one place
-5. If changes affect API defined by [oneMKL specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/) the design document must be reviewed by [UXL Foundation Math SIG forum](https://lists.uxlfoundation.org/g/Math-SIG) and contributed to [oneAPI specification](https://github.com/uxlfoundation/oneAPI-spec), and only after it the proposed changes can be implemented in this project.
+5. If changes affect API defined by [oneMKL specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/) the related part of the design document must be converted to oneAPI specification RFC (as a new [issue](https://github.com/uxlfoundation/oneAPI-spec/issues) with \[RFC\] tag), reviewed by [UXL Foundation Math SIG forum](https://lists.uxlfoundation.org/g/Math-SIG), and contributed to [oneAPI specification](https://github.com/uxlfoundation/oneAPI-spec), and only after it the proposed changes can be implemented in this project.
```
Comment (Rbiessy) [inline README.md]: I find that long sentence confusing and I'm not sure "converted" is the right word in this context, "duplicated" would be more truthful. I would suggest:

```suggestion
5. If the changes affect the API defined by [oneMKL specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/) a new [issue](https://github.com/uxlfoundation/oneAPI-spec/issues) with the \[RFC\] tag must be created to describe the API changes. The specification RFC must be reviewed by [UXL Foundation Math SIG forum](https://lists.uxlfoundation.org/g/Math-SIG), and only after it the proposed changes can be implemented in this project.
```

Comment (mkrainiuk) [reply README.md]: Agree, the suggestion looks better, thank you!

**Review by Rbiessy — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ contributors as reviewers to the PR.
...
     * It's recommended to keep all feedback as part of PR review, so it also
 will be documented in one place
-5. If changes affect API defined by [oneMKL specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/) the design document must be reviewed by [UXL Foundation Math SIG forum](https://lists.uxlfoundation.org/g/Math-SIG) and contributed to [oneAPI specification](https://github.com/uxlfoundation/oneAPI-spec), and only after it the proposed changes can be implemented in this project.
+5. If the changes affect the API defined by [oneMKL specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/) a new [issue](https://github.com/uxlfoundation/oneAPI-spec/issues) with the \[RFC\] tag must be created to describe the API changes. The specification RFC must be reviewed by [UXL Foundation Math SIG forum](https://lists.uxlfoundation.org/g/Math-SIG), and only after it the proposed changes can be implemented in this project.
```
Comment (dnhsieh-intel) [inline README.md]: I didn't find `RFC` in the current [labels](https://github.com/uxlfoundation/oneAPI-spec/labels). Will it be created? Or does it mean the issue title should be prefixed by "[RFC]"?

nitpicking
```suggestion
5. If the changes affect the API defined by [oneMKL specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/), a new [issue](https://github.com/uxlfoundation/oneAPI-spec/issues) with the \[RFC\] tag must be created to describe the API changes. The specification RFC must be reviewed by [UXL Foundation Math SIG forum](https://lists.uxlfoundation.org/g/Math-SIG), and only after it the proposed changes can be implemented in this project.
```

Comment (mkrainiuk) [reply README.md]: It's actually not a label, but a tag in the issue name https://github.com/uxlfoundation/oneAPI-spec/issues?q=is%3Aissue+is%3Aopen+RFC but maybe we can ask for the introducing the new label in the spec.

**Review by dnhsieh-intel — APPROVED** [review]

**Pull request merged**